schemas/kps/18.0/kps.xsd

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  .kps file schema, version 7.0
```

Comment:
Is this version 7.0 separate from version 18.0 in the README.md? 

Or is this to always match FileVersion 7.0?

Reply:
hmm, yeah, it should say 18.0

```suggestion
  .kps file schema, version 18.0
```